tests/unit/lms/services/canvas_api/factory_test.py

```diff
@@ -10,8 +10,13 @@
 
 
 class TestCanvasAPIClientFactory:
+    @pytest.mark.usefixtures("aes_service")
     def test_building_the_CanvasAPIClient(
-        self, pyramid_request, CanvasAPIClient, AuthenticatedClient, file_service
+        self,
+        pyramid_request,
+        CanvasAPIClient,
+        AuthenticatedClient,
+        file_service,
```

Comment:
```suggestion
        self, pyramid_request, CanvasAPIClient, AuthenticatedClient, file_service
```

If you snip the last comma off, Black will back this onto one line